src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Unix.Tests.cs

```diff
@@ -60,6 +60,7 @@ public void PropertiesOfInvalidDrive()
             Assert.Equal(invalidDriveName, invalidDrive.VolumeLabel);   // VolumeLabel is equivalent to Name on Unix
         }
 
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsSubsystemForLinux))] // https://github.com/dotnet/corefx/issues/11570
         [PlatformSpecific(TestPlatforms.AnyUnix)]
```

Comment:
Is this intentional that you want to skip this test for subsystem?

Reply:
yes, this test is still failing.